.github/workflows/e2e_nightly.yaml

```diff
@@ -27,7 +28,8 @@ jobs:
     uses: ./.github/workflows/_e2e_tests.yaml
     secrets: inherit
     with:
-      kic-image: kong/nightly-ingress-controller:nightly
+      kic-image: kong/nightly-ingress-controller
```

Comment:
This uses `kic-image` while the above we use `kic-image-repo`. This is confusing to use different parameter names for the same purpose IMO.

Reply:
updated all workflow files, please check again if there are inconsistencies in vars/envs and inputs/outputs.